client/src/components/Card/Card.js

```diff
@@ -0,0 +1,41 @@
+import styled from "styled-components/macro";
+import React from "react";
+import img from "../../assets/hubble-eyes.jpg";
+
+const Div = styled.div`
```

Comment:
This naming should be more specific
```suggestion
const CardContainer = styled.div`
```

Reply:
Thanks for suggestion 👍🏻